Cargo.toml

```diff
@@ -16,7 +16,7 @@ default = [ "reuse_port" ]
 reuse_port = []
 
 [dependencies]
-util = { package = "webrtc-util", version = "0.1.18" }
+util = { package = "webrtc-util", version = "0.1.18", default-features = false, features = ["error", "ifaces"] }
```

Comment:
What happens if removing 
default-features = false,
?

Reply:
in this case mdns package doesn't compile on armv5te because util package has "vnet" in default features list